components/kyma-operator/README.md

```diff
@@ -58,6 +58,16 @@ Execute the `run.sh` script with the `--skip-minikube-start` flag to rerun Kyma
   ./scripts/run.sh --skip-minikube-start
   ```
 
+## Upgrade Kyma
+
+The upgrade procedure relies heavily on Helm. When you trigger the upgrade, Helm performs `helm upgrade` on releases that exist in the cluster and are defined in the Kyma release you're upgrading to. If a Helm release is defined in the Kyma release you're upgrading to but is not present in the cluster when you trigger the upgrade, Helm performs `helm install` and installs such release.
```

Comment:
How 'bout using the term "Kyma version" instead of "Kyma release"? Maybe it's just me, but those numerous "releases" used in different contexts are pretty confusing.